api/v1beta2/ibmpowervscluster_types.go

```diff
@@ -32,26 +32,76 @@ const (
 
 // IBMPowerVSClusterSpec defines the desired state of IBMPowerVSCluster.
 type IBMPowerVSClusterSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
-
 	// ServiceInstanceID is the id of the power cloud instance where the vsi instance will get deployed.
 	// +kubebuilder:validation:MinLength=1
+	// Deprecated: use ServiceInstance instead
 	ServiceInstanceID string `json:"serviceInstanceID"`
 
 	// Network is the reference to the Network to use for this cluster.
+	// Whenf the field is omitted, A DHCP service will be created in the service instance and its private network will be used.
 	Network IBMPowerVSResourceReference `json:"network"`
 
 	// ControlPlaneEndpoint represents the endpoint used to communicate with the control plane.
 	// +optional
 	ControlPlaneEndpoint capiv1beta1.APIEndpoint `json:"controlPlaneEndpoint"`
+
+	// serviceInstance is the reference to the Power VS service on which the server instance(VM) will be created.
+	// Power VS service is a container for all Power VS instances at a specific geographic region.
+	// serviceInstance can be created via IBM Cloud catalog or CLI.
+	// supported serviceInstance identifier in PowerVSResource are Name and ID and that can be obtained from IBM Cloud UI or IBM Cloud cli.
+	// More detail about Power VS service instance.
+	// https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-creating-power-virtual-server
+	// when omitted system will dynamically create the service instance
+	// +optional
+	ServiceInstance IBMPowerVSResourceReference `json:"serviceInstance"`
+
+	// zone is the name of Power VS zone where the cluster will be created
+	Zone string `json:"zone"`
+
+	// resourceGroup name under which the resources will be created.
+	ResourceGroup string `json:"resourceGroup"`
+
+	// vpc contains information about IBM Cloud VPC resources
+	// +optional
+	VPC IBMVPCResourceReference `json:"vpc,omitempty"`
+
+	// transitGateway contains information about IBM Cloud TransitGateway.
+	// +optional
+	TransitGateway IBMTransitGatewayResource `json:"transitGateway,omitempty"`
+
+	// controlPlaneLoadBalancer is optional configuration for customizing control plane behavior.
```

Comment:
Might need to add info about that its a vpc load balancer like the comment in network